PatientWebApplication/PatientWebApplicationIntegrationTests/DoctorAppointmentControllerSetup.cs

```diff
@@ -0,0 +1,88 @@
+             200.0, false, "Cardiology", new List<DoctorNotification>(), "Ordination 2"), new List<Referral>(), "1");
+            DoctorAppointment appointment3 = new DoctorAppointment(3, new TimeSpan(0, 15, 0, 0, 0), "03/03/2020", new PatientUser(), new DoctorUser(2, "TestDoctorName2", "TestDoctorNameSurname2", "1234", "02/02/2020", "123", "email", "pass", "Grad",
+             200.0, false, "Cardiology", new List<DoctorNotification>(), "Ordination 2"), new List<Referral>(), "1");
+            DoctorAppointment appointment4 = new DoctorAppointment(4, new TimeSpan(0, 15, 45, 0, 0), "03/03/2020", new PatientUser(), new DoctorUser(2, "TestDoctorName2", "TestDoctorNameSurname2", "1234", "02/02/2020", "123", "email", "pass", "Grad",
+             200.0, false, "Cardiology", new List<DoctorNotification>(), "Ordination 2"), new List<Referral>(), "1");
+            DoctorAppointment appointment5 = new DoctorAppointment(5, new TimeSpan(0, 12, 0, 0, 0), "02/02/2020", new PatientUser(), new DoctorUser(1, "TestDoctorName1", "TestDoctorNameSurname1", "1234", "02/02/2020", "123", "email", "pass", "Grad",
+             200.0, false, "Cardiology", new List<DoctorNotification>(), "Ordination 2"), new List<Referral>(), "1");
+            DoctorAppointment appointment6 = new DoctorAppointment(6, new TimeSpan(0, 12, 15, 0, 0), "02/02/2020", new PatientUser(), new DoctorUser(3, "TestDoctorName3", "TestDoctorNameSurname3", "1234", "02/02/2020", "123", "email", "pass", "Grad",
+             200.0, false, "Cardiology", new List<DoctorNotification>(), "Ordination 2"), new List<Referral>(), "1");
+
+            DbContext.DoctorAppointments.Add(appointment1);
+            DbContext.DoctorAppointments.Add(appointment2);
+            DbContext.DoctorAppointments.Add(appointment3);
+            DbContext.DoctorAppointments.Add(appointment4);
+            DbContext.DoctorAppointments.Add(appointment5);
+            DbContext.DoctorAppointments.Add(appointment6);
+
+            OperationReferral operationReferral1 = new OperationReferral(1, "Medicine", "Take medicine until", 3, "classify", "comment", 1);
+            OperationReferral operationReferral2 = new OperationReferral(2, "Medicine2", "Take medicine until", 3, "Appointment", "comment", 2);
+
+            DbContext.OperationReferrals.Add(operationReferral1);
+            DbContext.OperationReferrals.Add(operationReferral2);
+
+            Operation operation1 = new Operation(1, 2, "03/03/2020", new TimeSpan(0, 14, 0, 0), new TimeSpan(0, 15, 0, 0, 0), 1, "room1");
+            Operation operation2 = new Operation(2, 1, "03/10/2020", new TimeSpan(0, 15, 0, 0), new TimeSpan(0, 15, 15, 0, 0), 2, "room1");
+
+            DbContext.Operations.Add(operation1);
+            DbContext.Operations.Add(operation2);
```

Comment:
Forward objects for classes directly as a parameter to the Add () method. For example:
"DbContext.<ObjectClass>.Add (new <ConstructorForClass>);"